conf/locale/eo/LC_MESSAGES/django.po

```diff
@@ -37,8 +37,8 @@ msgid ""
 msgstr ""
```

Comment:
These files (the `.po` files modified) shouldn't be checked in. From the [open edx internationalization docs](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#coverage-testing):  `This step creates new dummy translations in the Esperanto directory (edx-platform/conf/local/eo/LC_MESSAGES) and the RTL directory (edx-platform/conf/local/rtl/LC_MESSAGES). DO NOT CHECK THESE FILES IN. You should discard these files once you have finished testing.`